### ISSUE TITLE
Duplicated on Big Sur's now playing menu bar list

### ISSUE THREAD

**Issue by myungwoo** [steve228uk/YouTube-Music#106]
<img width="318" alt="image" src="https://user-images.githubusercontent.com/9026019/102493458-8f758000-40b6-11eb-86f2-e95d825b5ab0.png">

The YT Music app is duplicated on Big Sur's now playing menu bar list.
The first one does not show the album image, the title, and the artist's name. It can play and pause, however, skipping to the next song is unavailable.
The second one seems good, however, even if the song is playing, it shows the play button instead of the pause button.

Lastly, now playing in the control center shows up with the first one.
<img width="317" alt="image" src="https://user-images.githubusercontent.com/9026019/102493437-871d4500-40b6-11eb-8ec3-ffd834ef15cc.png">

**Comment by iswanj** [comment]
Yes this happens to me as well cannot go to next or previous song

**Comment by unbeatable-101** [comment]
> Yes this happens to me as well cannot go to next or previous song

That's because the version with a fix for the media key issue hasn't been released yet.

**Comment by abarisain** [comment]
Not sure if you can prevent WKWebView from contributing to macOS' media status, unfortunately

The pause button will be fixed by merging https://github.com/steve228uk/YouTube-Music/pull/48
